fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -3040,6 +3040,12 @@ public class Config extends ConfigBase {
     @ConfField
     public static String ssl_truststore_password = "";
 
+    /**
+     * Allow only encrypted connections from clients
+     **/
+    @ConfField
+    public static boolean require_secure_transport = false;
```

Comment:
Similar [config](https://dev.mysql.com/doc/refman/8.4/en/server-system-variables.html#sysvar_require_secure_transport) as in MySql server

Reply:
It is best to follow the naming convention of StarRocks, starting with ssl, and it is recommended to name it: ssl_force_secure_transport